flecsi/topo/index.hh

```diff
@@ -46,7 +46,7 @@ struct repartition : with_size, data::prefixes {
     update(sizes());
   }
 
-private:
+//private:
```

Comment:
Why does this need to change?

Reply:
The lambda defined inside the constructor accesses this function. MSVC was complaining about not being able to access a private member from inside it.

Reply:
That's… not how the language works.  I guess just conditionalize it on `_MSC_VER` or so?  Or just move the function into a `detail` namespace.

Reply:
> That's… not how the language works.

I know :/

> I guess just conditionalize it on _MSC_VER or so? Or just move the function into a detail namespace.

Will do.